src/Dapr.Actors/Runtime/ActorMethodContext.cs

```diff
@@ -40,6 +54,18 @@ public ActorCallType CallType
             get { return this.actorCallType; }
         }
 
+        /// <summary>
+        /// Gets the type of call by actor runtime (e.g. actor interface method, timer callback etc.).
```

Comment:
This is really right? It doesn't specify the type of call, it specifies the method being called.